services/distribution/src/test/java/app/coronawarn/server/services/distribution/assembly/component/DgcStructureProviderDefaultValuesetsMissingTest.java

```diff
@@ -20,6 +21,7 @@
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit.jupiter.SpringExtension;
 
+@Deprecated
```

Comment:
At what time would you delete the class? I am in favor or stating that explicitly, otherwise nobody will be bold enough to actually do it.